src/ApiCompat/Microsoft.DotNet.ApiCompat.Task/ValidatePackageTask.cs

```diff
@@ -124,22 +126,6 @@ public override bool Execute()
 
         protected override void ExecuteCore()
         {
-            Dictionary<string, string[]>? packageAssemblyReferences = null;
-
-            if (PackageAssemblyReferences != null)
-            {
-                packageAssemblyReferences = new Dictionary<string, string[]>(PackageAssemblyReferences.Length);
-                foreach (ITaskItem taskItem in PackageAssemblyReferences)
-                {
-                    string tfm = taskItem.GetMetadata("Identity");
-                    string? referencePath = taskItem.GetMetadata("ReferencePath");
-                    if (string.IsNullOrEmpty(referencePath))
-                        continue;
-
-                    packageAssemblyReferences.Add(tfm, referencePath.Split(','));
-                }
-            }
```

Comment:
This is dead and has been there for a year. No idea why I never realized that until now.